data.js

```diff
@@ -208,6 +208,41 @@ exports.data = [
             link: "https://www.wri.org/",
             label: "world-resource-institution",
         },
+        established: {
+            content: "1982",
+            updatedBy: "Andre", updatedAt: "2020-01-04",
+        },
+        kpis: {
+            content: "- Organizations using their guidelines \n- Contracts signed \n- Articles wrote \n- Press mentions",
+            updatedBy: "Andre", updatedAt: "2020-02-13",
+
+        },
+        description: {
+            content: "Another big global player fighting climate crisis. Their approach is to measure their work. Their motto is 'count it, change it, scale it'.",
+            updatedBy: "Andre", updatedAt: "2020-01-13",
+        },
+        
+        actions: {
+            content: "Examples of [2018](https://www.wri.org/annualreport/2018-19#page=4):"+
+                     "\n- Guidlines of WRI were adopted by several cities to develope a more sustainable traffic."+ 
```

Comment:
1. develop*e*
2. This is urban mobility, traffic is what you're stuck in
2. Suggestion `Creating guidelines for [what?] (adopted by [more precise, big, small, worldwide, European, ???] cities to make urban mobility more sustainable)`

Reply:
changed wording